index.md

```diff
@@ -10,4 +10,16 @@ show_call_box: true
 
 # Polish Quality Retreat
```

Comment:
I am not familiar with template so it is posible that maybe we should remove this header and instead bump all headers bellow by 1 level